lib/globalize/active_record/act_macro.rb

```diff
@@ -41,8 +41,13 @@ def allow_translation_of_attributes(attr_names)
 
         begin
           if database_connection_possible?
-            self.ignored_columns += translated_attribute_names.map(&:to_s)
-            reset_column_information
+            translated_attribute_name_strings = translated_attribute_names.map(&:to_s)
+            # Only ignore columns if they exist.  This allows queries to remain as .*
+            # instead of using explicit column names
+            unless (column_names & translated_attribute_name_strings).empty?
```

Comment:
```suggestion
            if (column_names & translated_attribute_name_strings).any?
```

I find this easier to read